src/styles/mixins.less

```diff
@@ -0,0 +1,17 @@
+// TODO: word-break mixins can be removed once the web console mixin is migrated to commons so these are accessible.
```

Comment:
Spoke with @rhamilto and created an issue to track items that will be shared. 
https://github.com/openshift/origin-web-console/issues/1500

I'll remove this mixin file.